README.md

```diff
@@ -481,22 +488,22 @@ do {
 
 ```swift
 // PREFERRED
-private static let kMyPrivateNumber: Int
+private static let myPrivateNumber: Int
 
 // NOT PREFERRED
-static private let kMyPrivateNumber: Int
+static private let myPrivateNumber: Int
 ```
 
 * **3.2.2** The access modifier keyword should not be on a line by itself - keep it inline with what it is describing.
```

Comment:
Hm, I agree that a rule about annotations might be useful, but it's not something new for Swift 3. We can talk about it separately.